### DISCUSSION TITLE
Update repository url: github -> bitbucket

### DISCUSSION THREAD

**Discussion by deepak-agendaboa** [codemagic-ci-cd/codemagic-docs#2061] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I moved our app from Github to Bitbucket. 
I want to keep the current configuration so I need to update the repository url of the app. But the field to edit the URL is disabled.

After checking other discussions, I have created a temp app in code magic from new URL.

Old one (need repository URL update) : https://codemagic.io/app/649d2c2f7eedd57afff80fbf

New one (temp app, will be deleted after URL update) : https://codemagic.io/app/650af90f561d061a2b6a77fc

### Expected results

Update the repository url.

### Actual results

Cannot do it myself.

**Comment by rafeysup93** [comment]
@deepak-agendaboa, I will certainly help you with that. May I know your Team/ Personal account ID for verification?

**Reply by rafeysup93** [reply, marked as answer]
Your app has been successfully moved. Please check and confirm.
